python/mlc_llm/chat_module.py

```diff
@@ -44,58 +45,61 @@ class ConvConfig:  # pylint: disable=too-many-instance-attributes
 
     Since the configuration is partial, everything will be ``Optional``.
 
+    The parameters are the same as :class:`mlc_llm.protocol.conversation_protocol.Conversation`
```

Comment:
Perhaps we should update https://llm.mlc.ai/docs/deploy/python.html#configure-mlcchat-in-python; from my understanding `ConvConfig` is not backward compatible, so the current one may fail after this PR.

Reply:
Right, i just updated the user docs